collections/_article/introducing-the-improved-ufbx-importer-in-godot-4-3.md

```diff
@@ -0,0 +1,57 @@
+
+In accordance with [our release policy](https://docs.godotengine.org/en/stable/about/release_policy.html#what-are-the-criteria-for-compatibility-across-engine-versions), we will continue to support projects which have chosen to use FBX2glTF throughout the 4.x cycle for compatibility with existing imports. ufbx will affect new 3d imports by default, but the importer for .fbx files may be changed in the Import dock.
+
+# Improved import of animated characters
+
+Those who have worked with .fbx in the past may have encountered a plethora of issues when importing skeletons and animations, to the extent that it became common practice to import such animation files in Blender and re-export as glTF 2.0. With Godot 4.3, we hope that this will be a thing of the past. We have made the following improvements to the skeleton importer:
+
+* [#89629 (Import rest pose as RESET animation)](https://github.com/godotengine/godot/pull/89629): This enables creating a RESET track to restore the skeleton to its imported pose. 
+* [#88824 (Retargeting option to use a template for silhouette)](https://github.com/godotengine/godot/pull/88824): This feature can be used to reference the RESET animation of a known good (T-pose) reference file. Having a separate reference pose .fbx is a common practice in some animation sets.
+* [#88821 (Allow preserving the initial bone pose in rest fixer)](https://github.com/godotengine/godot/pull/88821): This allows for importing a retargeted skeleton while keeping it in its initial pose.
+* [#88819 (Add new scene import option to import as Skeleton)](https://github.com/godotengine/godot/pull/88819): This solves cases especially common in .fbx files without any meshes, where animated skeletons import as a series of nodes.
+* Improvements to bone detection [#90050 (Add tips detection to auto mapping in bone mapper)](https://github.com/godotengine/godot/pull/90050): Improves support for Mixamo and ReadyPlayerMe finger bone conventions.
+* Several other bugfixes ([#90019](https://github.com/godotengine/godot/pull/90019), [#90064](https://github.com/godotengine/godot/pull/90064), [#90065](https://github.com/godotengine/godot/pull/90065), [#91641](https://github.com/godotengine/godot/pull/91641), [#92012](https://github.com/godotengine/godot/pull/92012)) to retargeting and animation import.
+
+# Issues and testing
+
+Samuli has spent exceptional time and effort testing the ufbx library on tens of thousands of .fbx test cases from a variety of sources, and we are proud to consider the new ufbx importer feature-complete. This level of rigorous design and testing is a stark contrast to previous .fbx importer rewrites some of you may recall from Godot 3.
```

Comment:
```suggestion
Samuli has spent exceptional time and effort testing the ufbx library on tens of thousands of .fbx test cases from a variety of sources, and we are proud to consider the new ufbx importer feature-complete. This level of rigorous design and testing should result in a much better experience for .fbx users.
```

The current phrasing feels a bit disrespectful to the people who have worked on the current FBX import support in 3.x. I think its better to just focus on the positives as it sends the same message across